src/webgpu/util/math.ts

```diff
@@ -668,12 +685,14 @@ const kInterestingF32Values: number[] = [
  * specific values of interest. If there are known values of interest they
  * should be appended to this list in the test generation code.
  */
-export function sparseF32Range(): Array<number> {
-  return kInterestingF32Values;
+export function sparseF32Range(finite_only: boolean): Array<number> {
```

Comment:
I now remember that typescript has string literals as enum-like types, so `boolean` could be `'full' | 'finite'`, but I can follow up with stylistic changes. No need to block.

Reply:
Thanks, maybe I'll do it tomorrow